glossary.md

```diff
@@ -111,10 +207,18 @@ Currently, available steps generated by PD include:
 
 [Partitioning](/partitioned-table.md) refers to physically dividing a table into smaller table partitions, which can be done by partition methods such as RANGE, LIST, HASH, and KEY partitioning.
 
+### PD
+
+Placement Driver (PD) is an important component of the [TiDB Architecture](/tidb-architecture.md#placement-driver-pd-server) that is responsible to store metadata and run the [TSO](/tso.md) that hands out timestamps that are used for transactions. It also orchestrates the data placement on TiKV and runs the [TiDB Dashboard](/dashboard/dashboard-overview.md).
```

Comment:
Not sure, it might. Maybe there are other things that I forgot about ("PD HTTP API"?). Does it have to be an exhaustive list? I think the current list has the most important tasks.

Reply:
The PD section of tidb-architecture.md where this links to also doesn't mention TiFlash.

Reply:
OK, thanks for checking!